python/desc/imsim/skyModel.py

```diff
@@ -87,46 +169,62 @@ def addNoiseAndBackground(self, image, bandpass=None, m5=None,
         @param [out] the input image with the background and noise model added to it.
         """
 
-        # calculate the sky background to be added to each pixel
+        #obs_db = 'minion_1016_sqlite_new_dithers.db'
+        #conn = sqlite3.connect(obs_db)
+        #c = conn.cursor()
+        #c.execute("SELECT obsHistID, moonPhase, dist2Moon, moonAlt FROM ObsHistory WHERE airmass>0")
+        #rows = c.fetchall()
+	
+	#obsid = 7177
+
+        #gen = ObservationMetaDataGenerator(database=obs_db, driver='sqlite')
+        #obs_md = gen.getObservationMetaData(obsHistID=rows[obsid][0], boundLength=3)[0]
+
+        camera = LsstSimMapper().camera
+        center_x, center_y = get_chip_center(chipName, camera)
+
+        
         skyModel = skybrightness.SkyModel(mags=True)
-        ra = np.array([self.obs_metadata.pointingRA])
-        dec = np.array([self.obs_metadata.pointingDec])
+        ra, dec = lsst.sims.coordUtils.raDecFromPixelCoords(xPix=center_x,
+                    yPix=center_y, chipName=chipName, camera=camera, 
+                    obs_metadata=self.obs_metadata, epoch=2000.0, includeDistortion=True)
         mjd = self.obs_metadata.mjd.TAI
-        skyModel.setRaDecMjd(ra, dec, mjd, degrees=True)
-
-        bandPassName = self.obs_metadata.bandpass
-        skyMagnitude = skyModel.returnMags()[bandPassName]
+        skyModel.setRaDecMjd([ra], [dec], mjd=mjd, degrees=True)
+         
+        bandPassName = self.obs_metadata.bandpass # obs_metadata returns y-band
```

Comment:
This comment, `# obs_metadata returns y-band`, isn't correct.  In this context, `self.obs_metadata.bandpass` should be set to whatever was specified in the input instance catalog.